### ISSUE TITLE
how to use the type resolution to make ForbiddenMethodCall to check bigdecimal.equals using bigdecimal == bigdecimal?

### ISSUE THREAD

**Issue by sazconaitbit** [detekt/detekt#3029]
Hi guys i want to make the ForbiddenMethodCall to prevent the use of bigDecimal.equals,

```
  ForbiddenMethodCall:
    active: true
    methods: ['kotlin.io.println', 'kotlin.io.print', 'kotlin.io.print', 'java.math.BigDecimal.equals'] 
```

running gradle detekMain works for bigDecimal.equals(bigdecimal) but in kotlin you use it as bigdecimal == bigdecimal so detekt don't work in this case using the == instead of .equals

I read that maybe I need to enable the type resolution?

checking in the doc website 

I need to add this:

`    withType<Detekt> {
        // Target version of the generated JVM bytecode. It is used for type resolution.
        this.jvmTarget = "1.8"
    }`

but in build.gradle.kts my IntelliJ Detekt is in red and also if i try to import the package detekt is not found

inside tasks {}

<img width="808" alt="Screen Shot 2020-08-28 at 03 36 05" src="https://user-images.githubusercontent.com/60626909/91529427-a877dc80-e8df-11ea-8b73-9b68361fb4d1.png">

I already have this line in 

plugins {
    id("io.gitlab.arturbosch.detekt") version "1.12.0"
}

Any ideas? Thanks.

**Comment by arturbosch** [comment]
Hi,
`detektMain` is already configured with type resolution.
The plain `detekt` task is not.
This is actually a false negative. The rule does not look at operator calls.
Thanks for reporting!
